tests/v2/validation/rbac/rbac.go

```diff
@@ -257,3 +259,78 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 	return
 
 }
+
+func waitForActiveCluster(client *rancher.Client, clusterID string) error {
+	err := kwait.Poll(500*time.Millisecond, 30*time.Minute, func() (done bool, err error) {
+		client, err = client.ReLogin()
+		if err != nil {
+			return false, err
+		}
+		clusterResp, err := client.Management.Cluster.ByID(clusterID)
+		if err != nil {
+			return false, err
+		}
+		if clusterResp.State == "active" {
+			return true, nil
+		}
+		return false, nil
+	})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func editCluster(client *rancher.Client, clustername string, namespace string, psact string) (clusterType string, err error) {
```

Comment:
can we change this `editCluster` name to include PSACT as well to be more specific on the actions we perform in the edit cluster?